### ISSUE TITLE
No graph definition files were found. 

### ISSUE THREAD

**Issue by catpanda** [tensorflow/tensorflow#11469]
hello I was learn to use the tensorflow, Now I can run softmax , nn , cnn and other model on it , this is a really good framework.
but now I suffered some problem.
I want to use the tensorboard, this is my code:
https://github.com/catpanda/tensorflow_demo/blob/master/minist/mnist_bp.py
you can see the result is very good. to celebrate that. I want to show them in graph.You see I use `writer = tf.summary.FileWriter('board/', graph=graph)` in line 56 and define the graph in line 53. then I run the tensorboard by `tensorboard --logdir='board/'`.

![qq 20170713125732](https://user-images.githubusercontent.com/6506928/28151076-040ad3f2-67cb-11e7-8f63-2cda654357ca.png)

but the page shows `No graph definition files were found. `。
![_2017-07-13t04-58-05 105z](https://user-images.githubusercontent.com/6506928/28151070-fd6f2746-67ca-11e7-8a45-1134e1dfecc6.png)

I use tensorflow on windows10 and by python3.5 and tensorflow1.2 I hope you can help me find out what is the problem.

**Comment by catpanda** [comment]
OK I finally solved it .
because the document has some problme that the dir did not need the ""

**Comment by jart** [comment]
Please ask questions like these on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) in the future. This issue tracker is for bugs and feature requests.

Out of curiosity, how did you solve your problem? What do you mean by empty string? I work on TensorBoard. What can we do to make it friendlier?

**Comment by catpanda** [comment]
I am sorry , because my mother laungue is not English, so I hard to read the express my question。
I suffered the problem because I can hardly understand the english document too. so I read the Chinese Document .In the Chine document we run the tensorboard by `tensorboard --logdir='dir'` but finally I found I shoud use  `tensorboard --logdir=dir` without quotation marks.

**Comment by jart** [comment]
That's peculiar because because the `tensorboard --logdir='dir'` should mean the same thing as `tensorboard --logdir=dir` if you're using bash and don't have any `$` in there.
